test/ws-sse/static/site-base.html

```diff
@@ -2,8 +2,8 @@
 <link rel="stylesheet" href="/stylesheet.css">
 <title>&lt;/&gt; htmx Realtime Test Server</title>
 <script src="/htmx/htmx.js"></script>
-<script src="/htmx/ext/sse.js"></script>
-<script src="/htmx/ext/ws.js"></script>
+<script src="https://unpkg.com/htmx-ext-sse@2.0.0/sse.js"></script>
```

Comment:
We're at version [2.2.1](https://github.com/bigskysoftware/htmx-extensions/blob/main/src/sse/package.json#L4) now

Maybe we should use `@latest` here? So we don't have to update the versions in this example as we release updates to these extensions in the future

Let me know what you think (objections welcome), otherwise at least let's use the latest version number here